templates/_layouts/_header.html

```diff
@@ -25,9 +25,10 @@
       </ul>
       {% if path.startswith('/docs') %}
       <form class="p-search-box" action="/docs/search">
-        <input type="search" id="search-docs" class="p-search-box__input" name="q" {% if query %}value="{{ query }}"{% endif %} placeholder="Search the docs" title="Search the documentation" autocomplete="on" required />
-        <button type="reset" id="search-docs-reset" class="p-search-box__reset u-no-margin--right" name="close"><i class="p-icon--close"></i></button>
-        <button type="submit" class="p-search-box__button" name="submit"><i class="p-icon--search"></i></button>
+        <label class="u-off-screen" for="search-docs">Search the docs</label>
```

Comment:
You can just put an `aria-label` attribute on the input instead of adding a hidden element.